map.py

```diff
@@ -91,30 +101,38 @@ def execute_user_move(user, new_column, new_row):
     user["column"] = new_column
     user["row"] = new_row
 
-def is_coordinate_passible(map, column, row):
-    if(map[column][row] == HH or map[column][row] == VV):
-        return False
-    elif(map[column][row] == EE):
+def is_coordinate_passable(map, column, row):
+    if(map[row][column] == EE):
         return True
+    else:
+        return False
 
 def can_player_move_to_coordinate(map, column, row):
     if(is_coordinate_on_map(map, column, row) == False): return False
-    if(is_coordinate_passible(map, column, row) == False): return False
+    if(is_coordinate_passable(map, column, row) == False): return False
 
     return True
 
 def game_board():
     while(True):
         draw_map(map_1, objects)
 
-        user_input = input("Use the 'k' and 'j' keys to move up and down.\nUse the 'h' and 'l' keys to move left and right.\n")
-        user["decision"] = map_parser.parse_user_move(user_input)
-        if(user["decision"] == "quit"): game_commands.quit()
-        if(user["decision"] == "cont"):
+        user_input = input("Use the 'k' and 'j' keys to move up and down.\nUse the 'h' and 'l' keys to move left and right.\nType 'quit' or 'q' to quit out of the game.\n")
+        user["m_decision"] = map_parser.parse_user_move(user_input)
+        if(user["m_decision"] == "quit"): game_commands.quit()
+        if(user["m_decision"] == "cont"):
             map_parser.show_controls()
             continue
 
-        new_column, new_row = determine_new_coordinates(map_1, user["decision"], user["column"], user["row"])
+        new_column, new_row = determine_new_coordinates(map_1, user["m_decision"], user["column"], user["row"])
         can_move = can_player_move_to_coordinate(map_1, new_column, new_row)
         if(can_move == True):
             execute_user_move(user, new_column, new_row)
+
+        npc_location = npc_coordinates(goblin)
```

Comment:
We probably want a better, more "data driven" way to deal with this NPC. Having that concept sort of floating about in your main loop isn't ideal.